src/services/BookService.tsx

```diff
@@ -43,5 +43,14 @@ export const fetchTopics = async () => client.get(`/catalog/topics`);
 export const postBook = async (book: NewBook) =>
   client.post(`/catalog/books`, book);
 
+export const getSubscription = async (bookId: number) =>
+  client.get(`/inventory/subscriptions/${bookId}`);
```

Comment:
We have mechanism in gateway of fetching all data for book details page. Getting subscription information should be done there, and then we wouldn't have getSubscription on our client